tachyon/crypto/commitments/kzg/kzg.h

```diff
@@ -75,6 +75,18 @@ class KZG {
     return g1_powers_of_tau_lagrange_;
   }
 
+#if TACHYON_CUDA
+  const device::gpu::GpuMemory<G1Point>& d_g1_powers_of_tau() const {
+    return d_g1_powers_of_tau_;
+  }
+
+  const device::gpu::GpuMemory<G1Point>& d_g1_powers_of_tau_lagrange() const {
+    return d_g1_powers_of_tau_lagrange_;
+  }
+
+  bool UsesGPU() const { return static_cast<bool>(msm_gpu_); }
```

Comment:
```suggestion
  bool UsesGPU() const { return msm_gpu_ != nullptr }
```

Reply:
I think it's find since this is a typical way to check whether the pointer of `unique_ptr<T>` isn't null.